package_mrtrix

```diff
@@ -72,7 +64,7 @@ EOF
 
     QTCORE=$(for n in $LIBS; do echo $n; done | grep 'libQt.*Core')
     [[ $QTCORE ]] && ( # sort out Qt:
-      QTPLUGDIR=$(strings $QTCORE | sed -n 's/qt_plugpath=\(.*\)$/\1/p')
+      QTPLUGDIR=/usr/lib/qt/plugins/ #$(strings $QTCORE | sed -n 's/qt_plugpath=\(.*\)$/\1/p')
```

Comment:
I'd make it an optional option and error out if it can't be detected automatically. The error message could suggest using `/usr/lib/qt/plugins/`. Given the target audience of the script that should suffice?